metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -218,7 +218,8 @@ def _handle_convert_column_evrs(  # noqa: C901 (complexity)
                 column_profile.uniqueProportion = res["observed_value"]
             elif exp == "expect_column_values_to_not_be_null":
                 column_profile.nullCount = res["unexpected_count"]
-                column_profile.nullProportion = res["unexpected_percent"] / 100
+                if res["unexpected_percent"]:
+                    column_profile.nullProportion = res["unexpected_percent"] / 100
```

Comment:
does this need an else? or the default is good?

Reply:
default is good. all fields are optional